### PR TITLE
Commerce 12719 2

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/COMMERCE-12719

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/type/deployer/CommerceCETDeployerImpl.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import org.osgi.framework.ServiceRegistration;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Andrea Sbarra
+ */
+@Component(service = CommerceCETDeployer.class)
+public class CommerceCETDeployerImpl implements CommerceCETDeployer {
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/type/deployer/CommerceCETDeployerImpl.java:34]: Looks like a generic commerce custom element deployer, but it is in fact specific for the custom checkout step type.
Shouldn't we better naming this then?
And if in the future we add other types of CET we can reuse the module and just create new deployers ecc

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]

**Comment by luca-pellizzon** [comment]
@andrea-ale-sbarra also I think you should better sort the commits

File: modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/type/deployer/CommerceCETDeployerImpl.java (hunk 1 above)
Comment (andrea-ale-sbarra) [reply modules/apps/commerce/commerce-client-extension-web/src/main/java/com/liferay/commerce/client/extension/web/internal/type/deployer/CommerceCETDeployerImpl.java:34]: it is true that now it only holds the checkout CX, but if we'd need to implement new CX types for Commerce we just need to add the cases in the deploy method, like it is done in the original CETDeployer

**Comment by andrea-ale-sbarra** [comment]
@luca-pellizzon you're right but if everything goes in before this PR, this PR will be of just one commit

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
no need to split anymore because dependent LPS is already in

**Pull request closed without merging**